src/Cassandra/IInternalStatement.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Cassandra.Data.Linq;
+using Cassandra.Mapping.Statements;
+
+namespace Cassandra
+{
+    internal interface IInternalStatement : IStatement
+    {
+        ITable GetTable();
```

Comment:
Fixed.
Also, maybe we can fix your `.DotSettings` file and persist this requirement. We need to add a single line like this one:
`<s:Int64 x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/BLANK_LINES_AROUND_SINGLE_LINE_INVOCABLE/@EntryValue">1</s:Int64>`
(I added it locally but doesn't commit because I think we can commit this fix into master right now without waiting for the approvement of this PR)